### PR TITLE
Couple logging additions

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs [hunk 1]
```diff
@@ -553,7 +553,7 @@ private void CheckDisposed()
                     }
                 }
             }
-            catch (Exception ex)
+            catch (Exception ex) when (!(ex is HubException))
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs]: When would this ever happen?

Comment (BrennanConroy) [reply src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs]: When you received a handshake response with an `"error"` set

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs]: That seems like a product bug no?

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs]: Ah I see, this code looks like it needs to be reworked, it's throwing and logging then rethrowing...

Comment (davidfowl) [reply src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs]: Add a comment please

File: clients/ts/signalr/src/Utils.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
```
Comment (davidfowl) [inline clients/ts/signalr/src/Utils.ts]: Isn't this just a helper file?

**Pull request merged**